src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable radix */
+/* eslint-disable no-unused-expressions */
```

Comment:
**Inline comment:** 
With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.